cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -367,7 +367,7 @@ def test_start_from_suspend(
 
 
 @pytest.mark.rhv3
-@pytest.mark.meta(blockers=[BZ(1496383, forced_streams=['5.9', '5.10', 'upstream'])])
```

Comment:
@lina-nikiforova I went  though BZ. which is interesting; It will be great, if you add it in `doc-string` for reference.